src/utils-lgpl/fcplay.c

```diff
@@ -347,6 +348,12 @@ void play_samples(char **files, unsigned int card, unsigned int device,
 		goto COMP_EXIT;
 	}
 
+	if (gapless) {
+		mdata.encoder_delay = 0;
+		mdata.encoder_padding = 0;
```

Comment:
memset mdata to 0?

Reply:
thanks for review,
ok, will add the memset.

Thanks,
mohs.